pkg/server/httpAPI.go

```diff
@@ -503,12 +507,39 @@ func (h *HTTPHandler) GetClusterState(ctx context.Context, clusterSize int, clie
 	return state
 }
 
-func getInitialCluster(ctx context.Context, initialCluster string, etcdConn brtypes.EtcdConnectionConfig, logger logrus.Entry, podName string) string {
-	// INITIAL_CLUSTER served via the etcd config must be tailored to the number of members in the cluster at that point. Else etcd complains with error "member count is unequal"
-	// One reason why we might want to have a strict ordering when members are joining the cluster
-	// addmember subcommand achieves this by making sure the pod with the previous index is running before attempting to add itself as a learner
+func getInitialCluster(ctx context.Context, initialCluster string, etcdConn brtypes.EtcdConnectionConfig, protocol string, clientPort string, logger logrus.Entry, podName string) string {
+	//INITIAL_CLUSTER served via the etcd config must be tailored to the number of members in the cluster at that point. Else etcd complains with error "member count is unequal"
+	//One reason why we might want to have a strict ordering when members are joining the cluster
+	//addmember subcommand achieves this by making sure the pod with the previous index is running before attempting to add itself as a learner
+
+	//Test for quorum loss
+	clientSet, err := miscellaneous.GetKubernetesClientSetOrError()
+	if err != nil {
+		logger.Errorf("failed to create clientset: %v", err)
```

Comment:
Isn't it is better to return error or fatal error here ?

Reply:
Done